propolis/src/hw/chipset/i440fx.rs

```diff
@@ -163,7 +164,19 @@ impl SelfArc for I440Fx {
         &self.sa_cell
     }
 }
-impl Entity for I440Fx {}
+impl Entity for I440Fx {
+    fn state_transition(
+        &self,
+        next: instance::State,
+        target: Option<instance::State>,
+        ctx: &DispCtx,
+    ) {
+        // TODO: make it easier to do this automatically
+        let bus = self.pci_bus.lock().unwrap();
+        let pm = bus.device_at(1, 3).unwrap().as_devinst().unwrap();
```

Comment:
Could we use a constant or something for these instead of magic numbers

Reply:
I do plan on a different method of access for these chipset owned devices, but until then, sure.